### DISCUSSION TITLE
multiple markers on a path

### DISCUSSION THREAD

**Discussion by darius-nck** [Leaflet/Leaflet#8996] [Q&A] [answered]
Hi there,
I am new to leaflet and was searching for a way, to have a path where markers can be added on, following it. But I could really find anything only plugins where I can put a path inbetween two markers but not a path "filled" with markers.
Any help would be appreciated.

best
Thomas

**Comment by IvanSanchez** [comment, marked as answer]
See https://github.com/bbecquet/Leaflet.PolylineDecorator  (which of course is in the plugins' list)

**Reply by darius-nck** [reply]
sorry, I havent seen this one, thanks a bunch!
